### PR TITLE
Restructure contrib directory

### PR DESCRIPTION
**Pull request by sumit-158**
Restructured the contrib unit test directory
fix: #769

### CODE REVIEW

**Comment by brettmc** [comment]
Hi @sumit-158 and welcome aboard! Can you please fix up the conflicts on your PR? Otherwise, it looks good to me!

**Comment by sumit-158** [comment]
Done! @brettmc

File: tests/Unit/Contrib/Prometheus/PrometheusExporterTest.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<?php
```
Comment (brettmc) [inline tests/Unit/Contrib/Prometheus/PrometheusExporterTest.php:1]: just FYI, the current metrics implementation is about to be replaced, so this test will disappear with it.

**Review by brettmc — APPROVED** [review]

**Comment by brettmc** [comment]
Last thing is can you fix the linting error? (`make all` locally will run all checks)

**Comment by sumit-158** [comment]
> 

Its failing with this error
image

**Comment by brettmc** [comment]
There are instructions in the readme on how to set up your dev environment (specifically, you need to set up your .env file)

**Comment by sumit-158** [comment]
Make all passes successfully locally

**Pull request merged**